app-shells/tasksh/tasksh-1.2.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Tasksh is a shell for Taskwarrior"
+HOMEPAGE="https://taskwarrior.org/"
+SRC_URI="https://taskwarrior.org/download/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+
+DEPEND="app-misc/task
+	sys-libs/readline:0"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	      cmake-utils_src_prepare
```

Comment:
Looks like a tab and a bunch of spaces here, and the below indented blocks are multiple tabs, let's be consistent with one tab per indentation?